google/datalab/contrib/bigquery/commands/_bigquery.py

```diff
@@ -112,7 +116,6 @@ def _get_load_parameters(bq_pipeline_input_config, bq_pipeline_transformation_co
 
     # The destination bigquery table name for loading
     source_of_table = bq_pipeline_input_config
-    load_task_config['table'] = source_of_table.get('table')
     if (load_task_config['table'] is None and not bq_pipeline_transformation_config and
```

Comment:
Your builds are failing due to this line, since load_task_config['table'] doesn't exist

Reply:
Yeah, I had tried cow-boy-resolving a merge-conflict with github's browser-based merge tool, so. Fixed now.